indexinglanguage/src/test/java/com/yahoo/vespa/indexinglanguage/linguistics/LinguisticsAnnotatorTestCase.java

```diff
@@ -90,7 +90,7 @@ public void requireThatTermAnnotationsAreLowerCased() {
                 if (!specialToken && !type.isIndexable()) {
                     continue;
                 }
-                assertAnnotations(expected, "foo", newToken("foo", "BAR", type, specialToken));
+                assertAnnotations(expected, "foo", newToken("foo", "bar", type, specialToken));
```

Comment:
please change this test to "requireThatTermAnnotationsAreNotLowerCased", keep "BAR" as input and check that the annotation contains BAR